packages/ra-ui-materialui/src/field/ReferenceArrayField.tsx

```diff
@@ -1,9 +1,28 @@
-import React, { Children, cloneElement, memo } from 'react';
+import React, { Children, cloneElement, FC, memo, ReactElement } from 'react';
 import PropTypes from 'prop-types';
 import LinearProgress from '@material-ui/core/LinearProgress';
 import { makeStyles } from '@material-ui/core/styles';
-import { useReferenceArrayFieldController } from 'ra-core';
+import {
+    useReferenceArrayFieldController,
+    Identifier,
+    RecordMap,
+} from 'ra-core';
 import { fieldPropTypes } from './types';
+import { ClassNameMap } from '@material-ui/styles';
+
+interface ReferenceArrayFieldProps {
```

Comment:
I think ra-ui-materialui should publish a generic FieldProps interface, which each individual field should extend.

Reply:
Agree. I wonder if the list here has included all generic props.

Reply:
Found `FieldProps` in `types.ts`, will use it to extend `ReferenceArrayFieldProps`